app/jobs/bulk_update_job.rb

```diff
@@ -19,6 +19,17 @@ def perform(ids:, args:)
 
   private
 
+    def resource_has_updateable_state?(resource)
+      (resource.state & blacklisted_states) == []
```

Comment:
Since we know there can only be one value in `resource.state`, why  not do `!blacklisted_states.include?(resource.state.first)`?

Reply:
I thought I should deal with the structure I had rather than assume what's in it. I can do that, though - I imagine we should eventually just make state single-valued.

Reply:
I appreciate the idea of dealing with the structure as it is — though I prefer to avoid infrequently used operators like this to avoid someone misreading it later.